Automattic/woocommerce-payments
# Avoid accidental pushes to `develop` and `trunk`

#### Changes proposed in this Pull Request
Adds additional confirmation step when pushing to protected branches to mitigate accidental pushes. 
More in paJDYF-cFD-p2

#### Testing instructions
* check out this branch
* update the `PROTECTED_BRANCH` variable so that it contains `avoid-accidental-push-to-develop`
* try to commit & push anything 
* confirm you're getting the warning and the script stops when input other than `y` is provided

-------------------


**Post merge**
